### DISCUSSION TITLE
Aliases for legacy scripts

### DISCUSSION THREAD

**Discussion by tonyjameshart** [Venafi/venafips#231] [Q&A] [answered]
Has anyone created aliases yet to give time to update scripts?

**Comment by tonyjameshart** [comment]
Aliases have been added where applicable.

**Comment by gdbarron** [comment, marked as answer]
Hi @tonyjameshart.  Not sure if you are asking and answering the question or not.  Function and parameter aliases were added where possible.  eg, in the case of -Venafi functions, it didn't make sense to add an alias since it could have been for either TLSPDC or TLSPC.  It is best to fully test your scripts prior to moving to v6.
